### PR TITLE
WIP: New queue

### PR DESCRIPTION
**Pull request by davidwaroquiers**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -24,8 +23,21 @@ classifiers = [
     "Topic :: Other/Nonlisted Topic",
     "Topic :: Scientific/Engineering",
 ]
-requires-python = ">=3.8"
-dependencies =[]
+requires-python = ">=3.9"
+dependencies =[
+    "jobflow[strict]",
```
Comment (Andrew-S-Rosen) [inline pyproject.toml]: Can the strict be dropped? Makes it difficult to test out in a real production python env.

**Review by Andrew-S-Rosen — COMMENTED** [review]
Looks exciting!

File: pyproject.toml (hunk 1 above)
Comment (davidwaroquiers) [reply pyproject.toml]: I think so but I'll let @gpetretto confirm this and make the change.

Comment (gpetretto) [reply pyproject.toml]: It was introduced during the pydantic v1 -> v2 transition to keep everything consistent. I don't think it is needed anymore and I removed it.

**Comment by davidwaroquiers** [comment]
I think this PR can be closed as all changes for the new queue have been merged into develop, and there is an open PR (#6 ) from develop to main to allow review of the code. @gpetretto

**Pull request closed without merging**